### PR TITLE
Switch to postprocess batch

### PR DESCRIPTION
**Pull request by John-WL**

### CODE REVIEW

File: lib_comfyui/polling_client.py [hunk 1]
```diff
@@ -20,11 +22,23 @@ class ComfyuiNodeWidgetRequests:
...
+        keep_waiting = True
+        while keep_waiting:
+            try:
+                result = cls.finished_comfyui_queue.get(timeout=1)
+                keep_waiting = False
+            except queue.Empty:
+                if getattr(shared_state, 'interrupted', False):
+                    return None
+
+        return result
```
Comment (John-WL) [inline lib_comfyui/polling_client.py:41]: do you think this is fine considering that we might not only use this "send" function for sending image generation requests?

File: scripts/comfyui.py [hunk 2]
```diff
@@ -49,45 +48,61 @@ def process(self, p, *args):
...
-        res.images = results
+        if batch_results is None or 'error' in batch_results:
+            return
 
-        self.save_image(results)
+        batch_size_factor = len(batch_results) // len(images)
+
+        if batch_number == 0:
+            lists_to_scale = [p.prompts, p.negative_prompts, p.seeds, p.subseeds,
+                              p.all_prompts, p.all_negative_prompts, p.all_seeds, p.all_subseeds]
```
Comment (ljleb) [inline scripts/comfyui.py:74]: This does not work as far as my testing goes. Rescaling `p.all_*` breaks generation and un-syncs the seeds.

To reproduce the issue, run a generation with batch size = 2, output nodes count = 3 and batch count = 5 (or any other combination of prime numbers)

Comment (John-WL) [reply scripts/comfyui.py:74]: how do we fix this then?

Comment (ljleb) [reply scripts/comfyui.py:74]: gotta patch create_infotext. See my PR for an implementation:
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/ca45ff1ae6fdd5c2dcd754fde95dd29f49bd414b/modules/processing.py#L720-L738

**Review by ljleb — APPROVED** [review]

**Pull request merged**